apis/github

```diff
@@ -96,22 +96,23 @@ Repository.tree = function(self, sha)
 	end
 	return __repoPriv[self].trees[sha]
 end
-Repository.cloneTo = function(self, dest, onProgress)
+Repository.cloneTo = function(self, treeName, dest, onProgress)
```

Comment:
This should definitely change to `Tree.cloneTo()`, since that way you don't need to pass in `treeName`. Another case of a poor design choice I made in the past